CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+
+## [0.43.2] - 2019-02-28
+### Updated
+- Upgraded plotly.js to 1.45.0 [#470](https://github.com/plotly/dash-core-components/pull/470)
+  - Patch releases [1.45.0](https://github.com/plotly/plotly.js/releases/tag/v1.45.0) containing numerous bug fixes
```

Comment:
1.45 is a feature release (but there *is* a patch release, 1.44.4, included in this update)
I'd suggest summarizing it as something like:
- Sankey diagram improvements including circular networks, node grouping, and concentration colorscales
- Matching cartesian axes
- Better bar, box, and violin alignment control
- Orthographic 3D projections
- Hovertemplate support for more trace types, including all 3D traces
- And many other features and bug fixes